### DISCUSSION TITLE
Dynamic database selection based on routes or folder names

### DISCUSSION THREAD

**Discussion by Naol-bm** [pocketbase/pocketbase#6147] [Q&A]
is it possible to make the database selection dynamic, e.g., based on the route or folder name, without needing to restart the app for each database?

for example, instead of running the app multiple times with commands like:

```
./pocketbase serve --dir="./pb_data1" --http="127.0.0.1:8091" &
./pocketbase serve --dir="./pb_data2" --http="127.0.0.1:8092" &
```

i want the app to dynamically select the database based on the route like 

> localhost:8090/db_name_or_whatever/_/

or local folder structure.

this would make it easier to manage multiple databases without spinning up separate app instances for each. any ideas or examples for implementing such a feature?

simple the goal is like this one

![Screenshot_2024-12-19-17-44-01_1920x1080](https://github.com/user-attachments/assets/b8fb4639-b429-4fa7-a0d3-3d49faf92e2b)

**Comment by Developermayank** [comment]
I also want this kind of feature.
